src/Pages/HomePage/HomePage.js

```diff
@@ -1,14 +1,52 @@
-import React from 'react'
```

Comment:
UI suggestion: the modal is overflowing
![Screenshot 2022-04-18 at 12 31 09 AM](https://user-images.githubusercontent.com/79623546/163728737-15c6e8cf-afbb-459c-bd76-99d992368fec.png)

Reply:
will fix it